src/software/ai/hl/stp/play/shoot_or_pass/shoot_or_pass_play_fsm.cpp

```diff
@@ -103,10 +103,6 @@ void ShootOrPassPlayFSM::startLookingForPass(const Update& event)
 
 void ShootOrPassPlayFSM::takePass(const Update& event)
 {
-    // Commit to a pass
-    LOG(DEBUG) << "Committing to pass: " << best_pass_and_score_so_far.pass;
```

Comment:
We would see this on the pass visualizer right?

Reply:
yeah thats correct, itll come through on the field widget